### PR TITLE
create an llvm flag

### PR DESCRIPTION
**Pull request by xmclark**
Some users may not care about building for the LLVM backend, so it may be useful to isolate it with a feature flag. Building should now only build the cranelift backend and does not require any LLVM setup.

### CODE REVIEW

**Comment by lachlansneff** [comment]
bors try

File: lib/emscripten/src/syscalls/windows.rs [hunk 1]
```diff
@@ -40,8 +40,8 @@ pub fn ___syscall5(ctx: &mut Ctx, which: c_int, mut varargs: VarArgs) -> c_int {
                 emscripten_memory_pointer!(ctx.memory(0), urandom_file_offset) as *const i8;
             let fd = unsafe { open(raw_pointer_to_urandom_file, flags, mode) };
             debug!(
-                "=> pathname: {}, flags: {}, mode: {} = fd: {}\npath: {}",
```
Comment (lachlansneff) [inline lib/emscripten/src/syscalls/windows.rs]: Did this stuff just get caught up in this pr?

**Comment by lachlansneff** [comment]
Okay, let's actually merge it
bors r+

File: lib/emscripten/src/syscalls/windows.rs (hunk 1 above)
Comment (xmclark) [reply lib/emscripten/src/syscalls/windows.rs]: yeah I have no idea how the build was passing before...makes me suspicious.

**Pull request merged**